test/HTML5/Parser/TokenizerTest.php

```diff
@@ -447,9 +447,9 @@ public function testSimpleTags()
         );
         foreach ($selfClose as $test => $expects) {
             $events = $this->parse($test);
-            $this->assertEquals(3, $events->depth(), "Counting events for '$test'" . print_r($events, true));
+            $this->assertEquals(2, $events->depth(), "Counting events for '$test'" . print_r($events, true));
             $this->assertEventEquals('startTag', $expects, $events->get(0));
-            $this->assertEventEquals('endTag', $expects, $events->get(1));
+            $this->assertTrue($events->get(0)['data'][2]);
```

Comment:
This syntax fails in PHP 5.3

https://travis-ci.org/Masterminds/html5-php/jobs/311341191#L260

Also, any chance this will ever get merged?